examples/platforms/nrf52840/platform.c

```diff
@@ -51,6 +51,16 @@ void __cxa_pure_virtual(void)
 
 void PlatformInit(int argc, char *argv[])
 {
+    extern bool gPlatformPseudoResetWasRequested;
+
+    if (gPlatformPseudoResetWasRequested)
+    {
+        nrf5RadioDeinit();
+        nrf5RadioInit();
```

Comment:
I'd consider reinitializing alarm as well, just to get it counting from 0 again (to have a valid timestamp in logs for example). I see though that our `otPlatAlarmInit` function does not initialize offsets to 0, so it could be added there as well.

Reply:
Done.